storage/remote/otlptranslator/prometheusremotewrite/helper.go

```diff
@@ -594,59 +520,37 @@ func addTypeAndUnitLabels(labels []prompb.Label, metadata prompb.MetricMetadata,
 	return labels
 }
 
-// getOrCreateTimeSeries returns the time series corresponding to the label set if existent, and false.
-// Otherwise it creates a new one and returns that, and true.
-func (c *PrometheusConverter) getOrCreateTimeSeries(lbls []prompb.Label) (*prompb.TimeSeries, bool) {
-	h := timeSeriesSignature(lbls)
+// timeSeriesIsNew returns true if the given labels are new, or false if they
+// already exist and stores the labels for future conflict checks.
+func (c *PrometheusConverter) timeSeriesIsNew(lbls labels.Labels) bool {
+	h := lbls.Hash()
```

Comment:
I'm not sure we actually need this function. I see it's used to check if the "_created" metric we generate is going to conflict with something else. But we want to get rid of that and we don't actually call this function for metrics that are NOT "_created" metrics, so it won't detect a conflict anyways. 

@ArthurSens did I understand correctly that we can remove this in favor of injecting CT zero sample ? Maybe we should do it in this PR.

Reply:
I'll open a PR to remove this, I think it will make things easier to review and you can rebase on top of it :)

Reply:
awesome, thank you

Reply:
https://github.com/prometheus/prometheus/pull/17014

Reply:
removed after merge from main in 84e9295a8f0d4460192c270954a68618f09c3d16